docs/data/material/getting-started/templates/templates.md

```diff
@@ -28,4 +29,6 @@ Please feel free to open an [issue](https://github.com/mui/material-ui/issues/ne
 
 Looking for something more? You can find complete templates and themes in the <a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store">premium template section</a>.
 
-<a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store"><img src="/static/images/themes-light.jpg" alt="react templates" width="2278" height="1358" /></a>
+<a href="https://mui.com/store/?utm_source=docs&utm_medium=referral&utm_campaign=templates-store">
+<img src="/static/images/themes-display.png" alt="react templates" width="1140" height="600" />
```

Comment:
Do we want to support dark mode as well? We could pull it off with

```
<div class="only-light-mode">
  <img src="/static/x/watermark-light.png" width="1306" height="536" style="width: 653px; margin-bottom: 2rem;" alt="" loading="lazy">
</div>
<div class="only-dark-mode">
  <img src="/static/x/watermark-dark.png" width="1290" height="548" style="width: 645px; margin-bottom: 2rem;" alt="" loading="lazy">
</div>
```

Reply:
Ohh! I'll explore this more — always wanted to have custom-tailored dark mode images!

Reply:
I'll leave that for a future PR where I can tackle light & dark images on the docs more holistically!
Will keep this one simple, for now 🤙